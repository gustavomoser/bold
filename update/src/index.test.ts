import * as Bold from './'

it('should export components to global namespace', () => {
  expect(Object.keys(Bold)).toMatchSnapshot()
})
