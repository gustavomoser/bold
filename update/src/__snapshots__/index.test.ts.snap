// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should export components to global namespace 1`] = `
[
  "Alert",
  "Breadcrumbs",
  "Button",
  "ButtonBase",
  "ButtonGroup",
  "Calendar",
  "MonthControl",
  "MonthView",
  "YearControl",
  "Checkbox",
  "Currency",
  "DateField",
  "DateInput",
  "DateRangePicker",
  "Dropdown",
  "DropdownDivider",
  "DropdownItem",
  "DropdownMenu",
  "FileUploader",
  "FormControl",
  "FormError",
  "FormLabel",
  "Cell",
  "Grid",
  "Heading",
  "HeadingSection",
  "HFlow",
  "Highlight",
  "Icon",
  "InfoLabel",
  "Input",
  "Link",
  "MaskedTextField",
  "Modal",
  "ModalBackdrop",
  "ModalBody",
  "ModalContainer",
  "ModalFooter",
  "ModalStore",
  "ModalMountTarget",
  "modal",
  "MonthField",
  "MonthPaginator",
  "MonthRangePicker",
  "MonthPicker",
  "Number",
  "Paginator",
  "Paper",
  "Portal",
  "Progress",
  "Radio",
  "RootRef",
  "SelectMenu",
  "SelectEmptyItem",
  "SelectHelperMenuItem",
  "SelectLoadingItem",
  "SelectMenuItem",
  "SelectDownshift",
  "defaultSelectFilter",
  "SelectSingle",
  "MultiDownshift",
  "SelectMulti",
  "SelectMultiInput",
  "SelectMultiItem",
  "Select",
  "SelectAsync",
  "SelectInline",
  "Spinner",
  "Step",
  "Stepper",
  "getStepProps",
  "useStepperState",
  "StickyContainer",
  "Switch",
  "DataTable",
  "DataTableDefault",
  "TableFilledBody",
  "TableLoadingRow",
  "TablePlaceholderRow",
  "PagedTable",
  "TableFooter",
  "Table",
  "TableBody",
  "TableCell",
  "TableHead",
  "TableHeader",
  "TableRow",
  "SortableLabel",
  "createStyles",
  "toggleDirection",
  "TabItem",
  "Tabs",
  "Tag",
  "Text",
  "TextArea",
  "TextField",
  "TextInput",
  "TextInputBase",
  "TimeField",
  "Tooltip",
  "VFlow",
  "LocaleContext",
  "useLocale",
  "DefaultGlobalCss",
  "GlobalCss",
  "colors",
  "useTheme",
  "useCss",
  "useStyles",
  "ThemeContext",
  "ThemeProvider",
  "breakpoints",
  "createBreakpoints",
  "createPallete",
  "createShadows",
  "createTheme",
  "focusBoxShadow",
  "createTypography",
  "radius",
  "darkTheme",
  "lightTheme",
  "zIndex",
  "hexToRGB",
  "invertColorScale",
  "debounce",
  "isEmpty",
  "isEqual",
  "merge",
  "some",
  "range",
  "clamp",
  "isPromise",
  "Observable",
  "__types",
]
`;
